src/audio/module_adapter/module_adapter.c

```diff
@@ -123,6 +123,20 @@ struct comp_dev *module_adapter_new(const struct comp_driver *drv,
 
 	dev->state = COMP_STATE_READY;
 
+#if CONFIG_IPC_MAJOR_4
+	struct sof_ipc_stream_params params;
+
+	/*
+	 * set the stream params based on the module base cfg. There's no need to initialize
```

Comment:
I think usually when we say, that a function "sets parameters," we mean, that we pass parameters to a function which are then used internally by that function for some configuration. While here as well as in `kpb_set_params()` above we **retrieve** parameters.